src/box/journal.h

```diff
@@ -101,6 +101,11 @@ struct journal_entry {
 	 * Approximate size of this request when encoded.
 	 */
 	size_t approx_len;
+	/**
+	 * Set to true when execution of a batch that contains this
+	 * journal entry is completed.
+	 */
+	bool complete;
```

Comment:
Please, try to name flags using a prefix like `is_`, `has_`, `does`, etc. Otherwise in context it might look like some other type.

Reply:
Done